docs/data/routes/docs/fundamentals/services/dictionary-service/en.md

```diff
@@ -44,14 +45,14 @@ const dictionaryService = new DictionaryService({
 ```
 The `DictionaryServiceConfig` type is found in the `@sitecore-jss\sitecore-jss` package. `DictionaryService` contains caching enabled by default.
 
-The `dataFetcher` option enables you to implement whichever data access method you wish. JSS ships with axios, which can be imported from `src\data-fetcher.js`.
+The `dataFetcher` option enables you to implement whichever data access method you wish. JSS ships with `axios`, which can be imported from `src\data-fetcher.js`.
```

Comment:
The dataFetcher thing is an older, obsolete pattern. At this point, only TrackingService needs it. There's a docs ticket in the current sprint where I provided more info about dataFetcher, let's discuss in our sync.